packages/snap/src/features/send/views/SendForm/events.tsx

```diff
@@ -64,15 +64,28 @@ async function onSourceAccountSelectorValueChange({
   event: InputChangeEvent;
   context: SendContext;
 }) {
-  context.fromAccountId = event.value as string;
+  const accountData = event.value as { accountId: string; addresses: string[] };
+  context.fromAccountId = accountData.accountId;
   context.error = null;
+
+  context.validation = {};
```

Comment:
why are we setting the validation to empty object? This will kill any other filed validation